com.unity.cinemachine/Samples~/3D Samples/Lock-on Target.unity

```diff
@@ -0,0 +1,1171 @@
+%YAML 1.1
```

Comment:
Using PositionComposer in Body causes jittery result. https://forum.unity.com/threads/target-lock-on-camera-issues.1350320/#post-8530517

I could not make a solution that was good with PositionComposer, because PositionComposer by its design lags behind a little bit, because it relies on the rotation of the camera.
Position based on rotation -> rotation changes because position changed -> so position needs to change again. This causes the jittery result in a locked on situation.

Reply:
hmmm... ok.  Leave the camera as is for now, but I still don't like it.
I think we need to address the jitter problems and fix them.  Even though the camera has the logic you described, there is no good reason for it to jitter, I think.

Reply:
I get no jitter at all in the build, with or without damping.  With 0 damping, there is still a little lag, but it is not jitter.  With damping, the lag is hidden.